test/signs_ui/alerts/state_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule SignsUi.Alerts.StateTest do
+  use ExUnit.Case, async: true
+  use SignsUiWeb.ChannelCase
+
+  describe "start_link" do
+    test "GenServer runs without crashing" do
+      @endpoint.subscribe("signs:all")
+
+      {:ok, _pid} = SignsUi.Alerts.State.start_link(interval_ms: 50)
+      Process.sleep(70)
```

Comment:
I don't think we should need this — `assert_broadcast` by default waits up to the ExUnit `assert_receive` timeout (which in turn defaults to 100ms) for the given message to be broadcasted. If we need to wait longer, we can also give a third argument to `assert_broadcast`.